src/shared/sleep-config.h

```diff
@@ -40,7 +40,7 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(SwapEntry*, swap_entry_free);
  * and the matching /proc/swap entry.
  */
 typedef struct HibernateLocation {
-        char *resume;
+        dev_t resume;
         uint64_t resume_offset;
```

Comment:
and just call this "offset"? the fact that this is about resuming is clear anyway, no?